README.md

```diff
@@ -69,19 +75,26 @@ err = p.Validate() // err: nil
 - `protoc` compiler in `$PATH`
 - `protoc-gen-validate` in `$PATH`
 - official language-specific plugin for target language(s)
-- **Only `proto3` syntax is currently supported.** `proto2` syntax support is planned.
+- **Only `proto3` syntax is currently supported.** `proto2` syntax support is
+  planned.
 
 ### Installation
 
 Installing PGV can currently only be done from source:
 
 ```sh
 # fetches this repo into $GOPATH
-go get -d github.com/envoyproxy/protoc-gen-validate
+go get -d github.com/bufbuild/protoc-gen-validate
 ```
 
-> #### 💡 Yes, our go module path is `github.com/envoyproxy/protoc-gen-validate` **not** `bufbuild` this is intentional.
-> Changing the module path is effectively creating a new, independent module.  We would prefer not to break our users. The Go team are working on better `cmd/go` support for modules that change paths, but progress is slow. Until then, we will continue to use the `envoyproxy` module path.
+> #### 💡 Yes, our go module path is `github.com/bufbuild/protoc-gen-validate` **
```

Comment:
our go module path is still `github.com/envoyproxy/protoc-gen-validate` I think?

Reply:
```suggestion
> #### 💡 Yes, our go module path is `github.com/envoyproxy/protoc-gen-validate` **
```